package.js

```diff
@@ -10,12 +10,12 @@ Package.onUse(function(api, where) {
   api.use([
     'mongo@1.1.2',
     'underscore@1.0.4',
-    'lai:collection-extensions@0.0.9',
-    'matb33:collection-hooks@0.8.1',
+    'lai:collection-extensions@0.2.1_1',
+    'matb33:collection-hooks@0.7.13',
```

Comment:
Wow forsome reason when i was looking yesterday it kept showing me the current version was 7.13 and i want confused about that since it was 8.1 in the package. TBH now that i looked again i have no idea. If you could update it back to 8.1 that would be great. Sorry about that.